src/components/Cart/cart.js

```diff
@@ -0,0 +1,76 @@
+import "../../styles.css";
+import { useCartAndWishListContext } from "../../contexts/cart-wishlist-context";
+
+export default function ({ cartItems }) {
+  const { state, dispatch  } = useCartAndWishListContext();
+
+  return (
+    <div>
+      {" "}
+      {state.itemsInCartArray.map((item) => (
+        <div
+          style={{
```

Comment:
You can extract the styles into a different file and add the className here instead of styles